src/alphabetical.py

```
@@ -0,0 +1,63 @@
+        sorted_lines.extend(sorted(list_items))
+
+    return sorted_lines, header_indices
+
+def main():
+    with open('README.md', 'r') as file:
+        lines = file.readlines()
+
+    # Convert names to title case
+    title_case_names = [convert_to_title_case(line) for line in lines]
+
+    sorted_lines, header_indices = sort_lists_alphabetically(title_case_names)
```

Ruff (0.13.1)

Unpacked variable `header_indices` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)